content/docs/command-reference/params/index.md

```diff
@@ -145,51 +146,38 @@ $ dvc run -n train -d logs/ -o users.csv \
 
 ## Examples: Python parameters file
 
-Consider this parameters file in Python format, named `params.py`:
+Consider this Python parameters file named `params.py`:
 
 ```python
-IS_BOOL: bool = True
-CONST = 5
-
-# All standard variable types are supported
```

Comment:
let's keep comments in this case serve as docs

Reply:
I didn't mean to remove that comment, my bad. Restored now.

Reply:
TBH it's not even ideal to have these notes inside the Python code example IMO but OK, we probably don't want to make a whole new section in the description above to explain all the rules around Python params files.

Reply:
>  whole new section in the description above to explain all the rules around Python params files.

that's what I would try to avoid... it's not scalable in this case, it's way easier to get all possibilities from the code to my mind

Reply:
Yeah. But shouldn't we have a section in https://dvc.org/doc/user-guide/dvc-files-and-directories to describe params files and the supported formats? We could have these notes there (and link from here).